web/book/src/transforms/window.md

```diff
@@ -84,13 +84,13 @@ You can also only apply `group`:
 from employees
 group department (
   sort age
-  derive rnk = rank
+  derive {rnk = rank age}
 )
 ```
 
 ## Window functions as first class citizens
 
-There is no limitaions where windowed expressions can be used:
+There is no limitations where windowed expressions can be used:
```

Comment:
```suggestion
There are no limitations on where windowed expressions can be used:
```

Reply:
@aljazerzen  why not accept these?

Reply:
I did accept them?

Reply:
Very sorry, I must have misread, and was thinking whether they were too refine-y...